site/docs/reference/Connectors/capture-connectors/salesforce.md

```diff
@@ -0,0 +1,135 @@
+# Salesforce
+
+This connector captures data from [Salesforce standard and custom objects](https://developer.salesforce.com/docs/atlas.en-us.238.0.object_reference.meta/object_reference/sforce_api_objects_concepts.htm) into Flow collections.
```

Comment:
At the moment we have limited the connector to only ingest these objects:

```
"Account",
"Contact",
"User",
"OpportunityFilledHistory",
"LeadHistory",
"Opportunity",
"Campaign",
"Case",
"ContactLineItem",
"Entitlement",
"Lead",
"LiveChatTranscript",
"MessagingSession",
"Quote",
"QuoteLineItem",
"ServiceAppointment",
"ServiceContract",
"Task",
"UserServicePresence",
"WorkOrder",
"WorkOrderLineItem"
```

So at the moment we don't support custom objects, and not all of the standard objects